client/context/query.go

```diff
@@ -309,7 +309,7 @@ func (ctx CLIContext) query(path string, key cmn.HexBytes) (res []byte, err erro
 		return res, errors.Errorf("query failed: (%d) %s", resp.Code, resp.Log)
 	}
 
-	// Data from trusted node or subspace query doesn't need verification
+	// data from trusted node or subspace query doesn't need verification
```

Comment:
Capitalize and add trailing period please.

Reply:
@HaoyangLiu I think he is stating that it should be reverted back to a sentence with a period.

Reply:
Got it! Thanks

Reply:
my pleasure 👍